### ISSUE TITLE
Can you share configs for training on other classes

### ISSUE THREAD

**Issue by v-iashin** [PeihaoChen/regnet#4]
Hi,

Thanks for sharing the RegNet weights for the dog class as well as the training config.

I could assume that all other classes are trained with the same config but the number of videos in some of them is significantly less (in AudioSet classes) than in others (classes from VEGAS).

This means that the number of iterations will be different and, therefore, learning rate and, thus, training dynamics.

I was wondering if you could share the RegNet training configs for other classes if not the pre-trained models.

**Comment by PeihaoChen** [comment]
Yes, there are a few differences between the configs of different sound types.

1. The number of training epochs for gun, hammer, sneeze, and cough should be changed to 4000, 10000, 10000, 10000, respectively considering the number of videos.
2. The testing set of gun, hammer, sneeze, and cough sound types includes the last 32 videos in each type as described in the paper. 

Other settings should be the same as the ./config/dog_opts.yml file.

**Comment by v-iashin** [comment]
Super. Thanks!

> number of training echo

what do you mean? “Epochs”?

**Comment by PeihaoChen** [comment]
Yes. Sorry for the typo.

**Comment by v-iashin** [comment]
Sure 🙂. 

Interesting decision! I am wondering why should we increase the number of epochs considering we have less data? Wouldn't it lead to overfitting? I am curious what’s you take on it?

Is it because it is related to the number of iterations, which, in turn, is related to the learning rate schedule or something of the kind?
